examples/mediatek/models/llm_models/weights/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Is it a wip folder?

Reply:
Mainly just to indicate to users where to put their model weights in.